src/api/background/handlers/alarms/gateway-update/gateway-update-alarm.handler.ts

```diff
@@ -22,22 +23,26 @@ export async function handleGatewayUpdateAlarm(alarm?: Alarms.Alarm) {
     return;
   }
 
-  let procData: ProcessedData[] = [];
+  let garItemsWithChecks: GatewayAddressRegistryItem[] = [];
 
   try {
     // fetch cache
     const ArIO = new AOProcess({ processId: AO_ARNS_PROCESS });
-    const gateways = (await ArIO.read({
+
+    const gatewaysResult = await ArIO.read<
+      PaginatedResult<GatewayAddressRegistryItemData>
```

Comment:
This result is paginated so the actual "gateways array" is nested and the previous enforced type was incorrect. Not sure why that might be, because I guess someone would have noticed if this never worked?